cmd/proxy/main.go

```diff
@@ -3,28 +3,52 @@ package main
 import (
 	"flag"
 	"net/http"
+	"os"
+	"strconv"
+	"strings"
 
 	log "github.com/sirupsen/logrus"
 
 	"slime.io/slime/modules/lazyload/pkg/proxy"
 )
 
+const WormholePorts = "wormholePorts"
+
 func main() {
+	// start health check server
 	go func() {
 		handler := &proxy.HealthzProxy{}
-		log.Println("Starting health check on :8080")
-		if err := http.ListenAndServe(":8080", handler); err != nil {
+		log.Println("Starting health check on :18080")
+		if err := http.ListenAndServe(":18080", handler); err != nil {
```

Comment:
no hardcode. pass via env/config and etc...

Reply:
add env "PROBE_PORT", default 8080